### ISSUE TITLE
Add ability to convert strings without writing to console

### ISSUE THREAD

**Issue by BD103** [tartley/colorama#311]
Creating a `convert(string: str)` global function would make certain programs easier.

I usually code Colorama into a Python package for Windows support. I find that I have to write an initialize script in `__init__.py` because it seems unnecessary to `init()` and `deinit()` every time. This convert function would fix that.

- It *wouldn't* rewrite sys.stdout / err
- It wouldn't be a part of a class, though it might implement AnsiToWin32
- It would only take one required argument, `string` (this leaves room for configuration)

Some examples of this used:

```python
from colorama import convert

def cli():
  print(convert("\033[31mHello, world!\033[0m"))

if __name__ == "__main__":
  cli()
```

**Comment by mara004** [comment]
Surely sounds like a good idea to me, but that would of course be a completely different, compatibility-breaking approach that would mean more or less a rewrite of colorama. I'm not sure whether a completely new project wouldn't be more suitable for this.

**Comment by BD103** [comment]
Yes. I just realized a few days ago that ANSI codes work on Windows 10, though I can't get them to work for the Python Repl. Do you know the escape pattern that would work?

**Comment by mara004** [comment]
ANSI codes working on the classical Windows 10 cmd is new to me. I thought that would require Windows-specific API calls?

**Comment by mara004** [comment]
> I find that I have to write an initialize script in `__init__.py` because it seems unnecessary to `init()` and `deinit()` every time. 

Yeah, putting the `colorama.init()` call in an `__init__.py` file is the right approach. Calling `init()` and `deinit()` repeatedly would be wasteful and is not how it's intended.
